CONTRIBUTING.md

```diff
@@ -76,34 +76,63 @@ Next you will have to set up the wrangler cli, so that you can connect to your c
   - `DISCORD_PUBLIC_KEY`
   - `DISCORD_TOKEN`
 
+- Encrypt all the variables after adding then save and deploy
+
 - Now, start the local server with the command `npm start`- make sure it is running on port `8787`
-- Open another terminal and type in the command `npm run ngrok`.
-- `ngrok` creates a secure tunnel that allows a local server to connect to external clients. It provides a URL that can be used to connect to a local server, just like if it were a public server hosted somewhere. For eg: Say you're running your app on `http://127.0.0.1:5501/` i.e localhost port 5501. Any external applications cannot connect to this server by default, but `ngrok` will give you a `http(s)` URL that any other client can use to connect to this server.
-- You will see 2 URLs generated, copy the `https` URL (eg: https://765m-321-132-44-44-44.ngrok.io)
+- Go to `cloudflare > workers-pages > copy the url under preview which endswith '.workers.dev' `
 - Now, go to [Discord Developer Portal](https://discord.com/developers/applications) and select your bot
-  - In the General Information, paste the link in the `INTERACTIONS ENDPOINT URL` field.
+  - In the General Information, paste the url in the `INTERACTIONS ENDPOINT URL` field.
 
 To verify if your bot is working:
 
 - Go to the server where your bot was invited
 - run a /hello command and the bot should reply with `Hello <Your_username>`
 
-Now add the public key in `rds-backend`
+Now to setup discord in `rds-backend`
+
+- Go to Your cloudflare `dashboard > workers > discord-slash-commands > settings > variables > edit Variables`
+- Now add one more variables to your environment:
 
-- Go to `local.js` in config
-- Create following there
+  - `RDS_SERVERLESS_PUBLIC_KEY`  ([generate here](https://cryptotools.net/rsagen))
+
+- Go to config/local.js, add
 
 ```
-botToken:{
-  botPublicKey:<Public key generated in the format similar to development.js>
-}
+ services:{
+   discordBot: {
+     baseUrl: "<Cloudflare_url>",
+   },
+ }
+
+ botToken: {
+   botPublicKey: "<Public key generated in the format similar to development.js in config>",
```

Comment:
What I meant was `similar to development.js` is misleading. We never edited the development.js file since the beginning.

Reply:
That s how it was mention earlier , and it is not misleading as we are talking about format. And it is clearly mentioned that we have change in local.js

Reply:
So why not change it to `local.js`? Why keep it `development.js`?

Reply:
we can't take reference from local.js